src/libraries/System.IO.MemoryMappedFiles/ref/System.IO.MemoryMappedFiles.cs

```diff
@@ -9,7 +9,6 @@ namespace Microsoft.Win32.SafeHandles
     public sealed partial class SafeMemoryMappedFileHandle : Microsoft.Win32.SafeHandles.SafeHandleZeroOrMinusOneIsInvalid
     {
         internal SafeMemoryMappedFileHandle() : base (default(bool)) { }
-        public override bool IsInvalid { get { throw null; } }
```

Comment:
Why is this being removed?

Reply:
Good point, I do see the implementation overriding it.  https://github.com/dotnet/runtime/blob/fd1cff8c26edbb7f0a59a52d12127a995c9ebd4d/src/libraries/System.IO.MemoryMappedFiles/src/Microsoft/Win32/SafeMemoryMappedFileHandle.Unix.cs#L83

Perhaps because it is `sealed` GenAPI thinks it doesn't need to be included?

Reply:
I think I saw this when doing the nullability work and it is because this is only overriden on `Unix` and not in `Windows`. So if you run GenAPI with the `Windows` asset it doesn't see the override. Maybe we should add it to a Manual file?

Reply:
Why not keep ref same and add to Windows impl?

Reply:
Yeah we can also do that and just call `base.IsInvalid`